content/docs/tasks/security/authz-permissive/index.md

```diff
@@ -0,0 +1,231 @@
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml@
+    logentry.config.istio.io/rbacsamplelog created
+    stdio.config.istio.io/rbacsamplehandler created
+    rule.config.istio.io/rabcsamplestdio created
+    {{< /text >}}
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should
+    see everything works fine.
+
+1.  Verify the logs stream has been created and check `permissiveResponseCode`.
+
+    In a Kubernetes environment, search through the `istio-telemetry`
+    pods' logs as follows:
+
+    {{< text bash json >}}
+    $ kubectl -n istio-system logs -l istio-mixer-type=telemetry -c mixer | grep \"instance\":\"rbacsamplelog.logentry.istio-system\"
+    {"level":"warn","time":"2018-08-30T21:53:42.059444Z","instance":"rbacsamplelog.logentry.istio-system","destination":"ratings","latency":"9.158879ms","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":48,"source":"reviews","user":"cluster.local/ns/default/sa/bookinfo-reviews"}
+    {"level":"warn","time":"2018-08-30T21:53:41.037824Z","instance":"rbacsamplelog.logentry.istio-system","destination":"reviews","latency":"1.091670916s","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":379,"source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {"level":"warn","time":"2018-08-30T21:53:41.019851Z","instance":"rbacsamplelog.logentry.istio-system","destination":"productpage","latency":"1.112521495s","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":5723,"source":"istio-ingressgateway","user":"cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"}
+    {{< /text >}}
+
+    In the above telemetry logs,  the `responseCode` is 200 which is what user see now.
+    The `permissiveResponseCode` is `denied` which is what user will see after switching
+    global authorization configuration from `PERMISSIVE` mode to `ENFORCED` mode, which
+    indicates the global authorization configuration will work as expected after rolling
+    to production.
+
+1.  Before rolling out a new authorization policy in production, apply it in permissive mode.
+    `Note`, when global authorization configuration is in permissive mode, all policies will be in
+    permissive mode by default.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/productpage-policy.yaml@
+    {{< /text >}}
+
+1.  Send traffic to the sample application again.
```

Comment:
Just copy the previous instance of this step. Avoid using "again" as it gives readers the impression that they are not making any progress.